source/core/ib_particle.cc

```diff
@@ -215,6 +215,41 @@ IBParticle<dim>::initialize_shape(const std::string         type,
                                                  position,
                                                  orientation);
     }
+  else if (type == "rbf")
+    {
+      if constexpr (dim == 3)
```

Comment:
Also why would this be only supported for dim==3?

Reply:
The bitpit library doesn't allow 2D for all their objects, and that's the case for one that is used in the RBF generation script. At the moment the script can't produce 2D objects, but in theory this limitation wouldn't interfere Lethe. I thought we shouldn't allow using 2D RBF shapes since we can't produce them, but I can allow them if you think it would be pertinent.